developer-docs-site/docs/guides/resource-accounts/common-questions.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "Common questions"
+id: "common-questions"
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Common questions
+
+## What is a resource account?
+
+A [resource account](../../move/move-on-aptos/resource-accounts.md) is an [Account](../../concepts/accounts/) that's used to store and manage resources independent of a user. It can be used as a simple storage account or it can be utilized to programmatically manage resources from within a smart contract. 
+
+## How are resource accounts created?
+
+Let's review the two functions used to create resource accounts and what they return.
+
+First off, note that both creation functions allow for the input of a `seed` byte vector for the [ensuing hash used to compute the resource address](#how-is-the-address-for-a-resource-account-derived).
```

Comment:
maybe a brief explanation on what is a seed?

Reply:
Done!